great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -1711,11 +1748,10 @@ def _chart_domain_values(
             ]
         ] = None
         chart: Optional[alt.Chart] = None
-        if plot_mode is PlotMode.PRESCRIPTIVE:
-            if metric_name in implemented_metrics:
+        if metric_name in implemented_metrics:
+            if plot_mode is PlotMode.PRESCRIPTIVE:
                 plot_impl = self.get_expect_domain_values_to_be_between_chart
-        elif plot_mode is PlotMode.DESCRIPTIVE:
-            if metric_name in implemented_metrics:
+            elif plot_mode is PlotMode.DESCRIPTIVE:
```

Comment:
Flip around conditionals to save on some code (don't need to use `if metric_name in implemented_metrics` both times).